devimint/src/main.rs

```diff
@@ -1175,6 +1174,18 @@ async fn main() -> Result<()> {
     Ok(())
 }
 
+#[tokio::main]
+async fn main() -> Result<()> {
+    let ready_file = PathBuf::from(env::var("FM_TEST_DIR")?).join("ready");
+    match handle_command().await {
+        Ok(r) => Ok(r),
+        Err(e) => {
+            write_overwrite_async(ready_file, "ERROR").await?;
+            Err(e)
+        }
+    }
+}
+
```

Comment:
If devimint returned an error, I think it wasn't writing error status to that "ready file" to inform mprocs